rectools/models/nn/transformers/lightning.py

```diff
@@ -148,16 +164,20 @@ def on_train_start(self) -> None:
     def training_step(self, batch: tp.Dict[str, torch.Tensor], batch_idx: int) -> torch.Tensor:
         """Training step."""
         x, y, w = batch["x"], batch["y"], batch["yw"]
+
+        if self.requires_negatives:
```

Comment:
I think we can move this common logic in train and val steps to a separate method `get_logits(batch)` after `get_loss_calculator` is introduced.

Reply:
I made `get_loss_calculator`, `get_bacth_logits(batch)` and changed the default value of n_negatives to 0.